js/components/lab-model.js

```diff
@@ -20,8 +20,10 @@ export default class LabModel extends PureComponent {
   }
 
   componentDidUpdate(prevProps) {
-    if (this.props.model !== prevProps.model) {
-      this.setState({loading: true});
+    if (this.props.model !== prevProps.model && !this.props.liveData) {
+      this.setState({ loading: true });
+    } else {
+      this.setState({ loading: false });
```

Comment:
What was purpose of this change? `loading` state is used to display loading icon when we switch models (as it can take a while). Why is this disabled when sensor is used?